### DISCUSSION TITLE
How to register the web wallet to Rainbow kit.

### DISCUSSION THREAD

**Discussion by imduchuyyy** [rainbow-me/rainbowkit#2090] [Q&A] [answered]
We built a web wallet and want to register to the rainbow kit. How to do it ?

**Comment by DanielSinclair** [comment, marked as answer]
RainbowKit does not currently support most types of Web Wallets, but you can create a Custom Wallet connector for your dApp. If the wallet SDK is supported upstream in Wagmi, we could introduce this to RainbowKit more broadly as well. https://www.rainbowkit.com/docs/custom-wallets
